prboom2/src/m_file.c

```diff
@@ -545,3 +546,26 @@ char *M_getenv(const char *name) {
   return getenv(name);
 #endif
 }
+
+dboolean M_ClearDirectory(const char *path)
```

Comment:
This is non-recursive right? Maybe there is a more specific name. `M_RemoveFilesAtPath`?